api/src/utils/apply-query.ts

```diff
@@ -805,11 +805,13 @@ export function applyFilter(
 }
 
 export async function applySearch(
+	knex: Knex,
```

Comment:
Do we have to pass in 2 Knex instances to this function? 🤔

Reply:
Isn't one of them the current query we're building? I just followed the spirit of the `applyFilter` function that also receives the Knex instance and the current QueryBuilder

Reply:
Makes sense 😄